src/components/SingleEvent/Admin/PartyForm.js

```diff
@@ -9,8 +11,10 @@ class PartyForm extends Component {
       name = '',
       description = '',
       location = '',
-      date = '',
-      image = '',
+      start = undefined,
```

Comment:
When I start up, I am getting `Error: Invalid date: Invalid Date`.  I think it's because passing empty string or undefined to `new Date()` will return `Invalid Date`. Are we supposed to put some default date here ?

When I changed to the following the error disappeared.

```
makoto@Makotos-MacBook-Air: [~/work/noblocknoparty/app -  (sql)] $ git diff
diff --git a/src/components/SingleEvent/Admin/PartyForm.js b/src/components/SingleEvent/Admin/PartyForm.js
index 8fff79e..00f5647 100644
--- a/src/components/SingleEvent/Admin/PartyForm.js
+++ b/src/components/SingleEvent/Admin/PartyForm.js
@@ -11,9 +11,9 @@ class PartyForm extends Component {
       name = '',
       description = '',
       location = '',
-      start = undefined,
-      end = undefined,
-      arriveBy = undefined,
+      start = new Date(),
+      end = new Date(),
+      arriveBy = new Date(),
       headerImg = '',
       deposit = '0.02',
       coolingPeriod = `${60 * 60 * 24 * 7}`,
@@ -64,7 +64,7 @@ class PartyForm extends Component {
```

Reply:
Strange that it does that, but thanks for figuring it out.